docs/docs/whatsnew.md

```diff
@@ -41,3 +41,23 @@ If you want to use the old `mvn -B package` command instead of the `maven` build
 ```
 
 The same approach can be implemented if the `maven` builder is used in the global build step.
+
+## v1.2.25
+The `build-parameters ignore` attribute will be upgraded to support [full glob pattern](https://en.wikipedia.org/wiki/Glob_(programming)). Wildcards, such as `*`,`!`,`?` can be used to match characters. 
```

Comment:
The `build-parameters ignore` attribute will be upgraded to support [full glob patterns](https://en.wikipedia.org/wiki/Glob_(programming)).

Reply:
OK, done